### ISSUE TITLE
simglm - example code for simulating non-normal nested data

### ISSUE THREAD

**Issue by lpp22** [lebebr01/simglm#101]
HI.  The below code from the vignette only produces an outcome with all 1's instead of a distribution of 0's and 1's.  Is there an error in the code?

set.seed(321) 

sim_arguments <- list(
  formula = y ~ 1 + weight + age + sex,
  fixed = list(weight = list(var_type = 'continuous', mean = 180, sd = 30),
               age = list(var_type = 'ordinal', levels = 30:60),
               sex = list(var_type = 'factor', levels = c('male', 'female'))),
  error = list(variance = 25),
  sample_size = 10,
  reg_weights = c(2, 0.3, -0.1, 0.5),
  outcome_type = 'binary'
)

simulate_fixed(data = NULL, sim_arguments) %>%
  simulate_error(sim_arguments) %>%
  generate_response(sim_arguments)

**Comment by lebebr01** [comment]
Thank you for reaching out and interest in the package. This is not an error in the code, but rather a poor choice I made for the regression weights in this example. I also generated data that are not centered which make the intercept not interpretable. I've included some code below that generates both 0/1 values by changing the regression weights to more plausible values. 

```r
set.seed(321)

sim_arguments <- list(
    formula = y ~ 1 + weight + age + sex,
    fixed = list(weight = list(var_type = 'continuous', mean = 180, sd = 30),
                 age = list(var_type = 'ordinal', levels = 30:60),
                 sex = list(var_type = 'factor', levels = c('male', 'female'))),
    error = list(variance = 25),
    sample_size = 10,
    reg_weights = c(-5, 0.03, -0.01, 0.5),
    outcome_type = 'binary'
)

simulate_fixed(data = NULL, sim_arguments) %>%
    simulate_error(sim_arguments) %>%
    generate_response(sim_arguments)
```

**Comment by lpp22** [comment]
Thank you so much. This is very helpful.

Sent from my iPhone

On Dec 7, 2022, at 10:40 AM, Brandon LeBeau ***@***.***> wrote:

﻿

External.

Closed #101<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flebebr01%2Fsimglm%2Fissues%2F101&data=05%7C01%7Clpp22%40drexel.edu%7Cb28e0bc11530431e4b1b08dad86964b8%7C3664e6fa47bd45a696708c4f080f8ca6%7C0%7C0%7C638060244455039330%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6IsQ7qc0QqsZaaR%2Fhqg%2F0tnGHdH7FDbw%2BT6RfsCc0ho%3D&reserved=0> as completed.

**Comment by lpp22** [comment]
Thank you again. I will try these fixes.

Sent from my iPhone

On Dec 7, 2022, at 10:40 AM, Brandon LeBeau ***@***.***> wrote:

﻿

External.

Closed #101<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flebebr01%2Fsimglm%2Fissues%2F101&data=05%7C01%7Clpp22%40drexel.edu%7Cb28e0bc11530431e4b1b08dad86964b8%7C3664e6fa47bd45a696708c4f080f8ca6%7C0%7C0%7C638060244455039330%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6IsQ7qc0QqsZaaR%2Fhqg%2F0tnGHdH7FDbw%2BT6RfsCc0ho%3D&reserved=0> as completed.
